site_scons/site_tools/compiler_setup.py

```diff
@@ -54,10 +54,14 @@ def _base_setup(env):
                         '-fpic'])
 
     env.AppendIfSupported(CCFLAGS=DESIRED_FLAGS)
-
+    env.Append(CCFLAGS=['-Wno-unused-function'])
```

Comment:
do we want to disable these globally rather than with some pragma, either _Pragma in macros or #pragma in C code.  Examples abound.